### DISCUSSION TITLE
Add support for traffic global's TCAS api

### DISCUSSION THREAD

**Discussion by Phil787-9** [magknight/787aviatorsTracker#818] [Ideas]
**Describe the bug**
Traffic injected by Traffic Global does no longer show on the MK ND, despite TCAS-setting enabled in TG and Traffic enabled on the ND of the MK.

**To Reproduce**
Set up the MK ND to show traffic and enable the "TCAS"-setting in TG

**Expected behaviour**
Traffic should show on the ND, as it was the case in previous versions (e.g. 1.7.18 --> pre-Custom ND)

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Windows 11
 - X-Plane version: 11.55
 - Version: 1.7.49

**Additional context**
Problem did not persist prior to the custom ND (e.g. under 1.7.18). Haven't tracked when exactly traffic injection stopped working because I skipped a couple of versions here and there.

**Comment by aeroniemi** [comment]
the traffic global manual implies that they don't support the 11.50 tech, and instead have their own stuff. we're unlikely to add this unless there is significant interest as it looks like it'd be quite a large amount of work for limited reward
